### PR TITLE
port_bounce.py test enhancement

### PR DESCRIPTION
**Pull request by Naresh-ibm**
enahnaced the port_bounce.py test by adding the multipath verification.
removed the port_number dependency of passing it through yaml.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
linux-7sxk:~/nani # avocado run port_bounce_latest.py -m /root/port_bounce.py.data/port_bounce.yaml
JOB ID     : fb4cf19c1da50dc0c11975625f67058c99e3fc3b
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-19T02.29-fb4cf19/job.log
 (1/1) port_bounce_latest.py:PortBounceTest.test;run-6b9e: PASS (1036.88 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1036.97 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-19T02.29-fb4cf19/results.html
linux-7sxk:~/nani #

**Comment by Naresh-ibm** [comment]
[port_bounce_lates_new_job_log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3868035/port_bounce_lates_new_job_log.txt)

File: io/disk/port_bounce.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        defins and test for different scenarios
+        '''
+        bounce_time = [self.sbt, self.lbt]
+        self.log.info("short/long port bounce for individual ports:")
+        for b_time in bounce_time:
+            for port in self.port_ids:
+                test_ports = []
+                test_ports.append(port)
+                for _ in range(self.count):
+                    self.porttoggle(test_ports, b_time)
```
Comment (narasimhan-v) [inline io/disk/port_bounce.py]: @Naresh-ibm can you explain what you are trying to do here ?

File: io/disk/port_bounce.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+        self.log.info("port bounce for all ports:")
+        for _ in range(self.count):
+            self.porttoggle(self.port_ids, 300)
+            time.sleep(20)
```
Comment (narasimhan-v) [inline io/disk/port_bounce.py]: Can this 20 not be `self.verify_sleep_time` ?

**Pull request closed without merging**